_data/cryptocurrencies.yml

```
@@ -187,6 +187,14 @@ websites:
       - co
       - pe
 
+  - name: BuyCoins
+    url: https://buycoins.africa/
+    img: buycoins.svg
+    tfa:
+      - sms
+      - totp
+    doc: https://help.buycoins.africa/article/9y6tnkdn9h
+
```

```suggestion
    regions:
      - ng

```

According to the [terms and conditions](https://buycoins.africa/terms/):

> Your eligibility to access certain BuyCoins services also depends on the country in which you reside. At the moment, BuyCoins operates in Nigeria, and is only available to users who hold Nigerian bank accounts.